app/src/ui/updates/update-available.tsx

```diff
@@ -25,7 +25,10 @@ export class UpdateAvailable extends React.Component<
           An updated version of GitHub Desktop is available and will be
           installed at the next launch. See{' '}
           <LinkButton uri={this.props.releaseNotesLink}>what's new</LinkButton>{' '}
-          or <LinkButton onClick={this.updateNow}>restart now</LinkButton>.
+          or{' '}
+          <LinkButton onClick={this.updateNow}>
+            restart GitHub Desktop now
```

Comment:
Just porting over @donokuda's comment from #4941 

> I don't think the "now" is necessary (although I don't strongly feel either way.)

I'm also 👍 to dropping this, but these are not strong feels